### PR TITLE
Extra checks when pushing an image to gcr repository

### PR DESCRIPTION
**Pull request by mwielgus**
Adds:
* Confirmation of pushed image.
* Check whether the image is already there.

### CODE REVIEW

File: cluster-autoscaler/push_image.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#/bin/bash
+# Copyright YEAR The Kubernetes Authors.
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/push_image.sh]: 2017?

File: cluster-autoscaler/push_image.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMAGE_TO_PUSH=$1
+if [ -z $IMAGE_TO_PUSH ]; then
+  echo No image passed
+  exit 1
+fi
+
+echo About to push image $IMAGE_TO_PUSH
```
Comment (MaciekPytel) [inline cluster-autoscaler/push_image.sh]: Why no ""?

File: cluster-autoscaler/push_image.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#/bin/bash
+# Copyright 2017 The Kubernetes Authors.
```
Comment (MaciekPytel) [inline cluster-autoscaler/push_image.sh]: I think you need a newline after /bin/bash for boilerplate to pass

**Pull request merged**